api/datalake_api/settings.py

```diff
@@ -11,10 +11,16 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations under
 # the License.
+import os
 
 # default settings
+
 DYNAMODB_TABLE = 'test'
 DYNAMODB_LATEST_TABLE = 'test_latest'
+DATALAKE_USE_LATEST_TABLE = \
+    os.environ.get("DATALAKE_USE_LATEST_TABLE", "false").lower() == "true"
+LATEST_MAX_LOOKBACK = int(os.environ.get("LATEST_MAX_LOOKBACK", "30"))
```

Comment:
I think the default needs to be like at least 10000, since there are production users that have it set to 9999.